src/component-mocks.ts

```diff
@@ -53,8 +53,10 @@ jest.mock('@zero-tech/zui/components', () => ({
   Tabs: () => null,
   TabsNav: () => null,
   TextStack: () => null,
+  ToastNotification: () => null,
```

Comment:
I regenerated the mock files for the zUI components and icons as I noticed we hadn't done it in a while and there were a couple of new ones. I can split this out to a separate PR if preferred.